src/spec.c

```diff
@@ -1246,16 +1247,35 @@ void CleanPool_ThreadPoolDestroy() {
   }
 }
 
-size_t CleanPool_WorkingThreadCount() {
-  if (cleanPool) {
-    return redisearch_thpool_num_threads_working(cleanPool);
-  }
+size_t getPendingIndexDrop() {
+  return __atomic_load_n(&pendingIndexDropCount_g, __ATOMIC_RELAXED);
+}
+
+size_t addPendingIndexDrop() {
+  __atomic_fetch_add(&pendingIndexDropCount_g, 1, __ATOMIC_RELAXED);
```

Comment:
This will return the number before the addition.
```suggestion
  __atomic_add_fetch(&pendingIndexDropCount_g, 1, __ATOMIC_RELAXED);
```